src/main/java/com/nitnelave/CreeperHeal/block/CreeperBanner.java

```diff
@@ -20,20 +23,21 @@ protected CreeperBanner(Banner banner)
     }
 
     /*
-     * (non-Javadoc)
-     * 
-     * @see com.nitnelave.CreeperHeal.block.CreeperBlock#update()
+     * @see com.nitnelave.CreeperHeal.block.Replaceable#drop(boolean)
      */
     @Override
-    public void update()
+    public boolean drop(boolean forced)
     {
-        super.update();
-        Banner state = (Banner) getBlock().getState();
-        Banner banner = (Banner) blockState;
-        state.setBaseColor(banner.getBaseColor());
-        state.setPatterns(banner.getPatterns());
-
-        state.setData(banner.getData());
-        state.update(true);
+        if (forced || CreeperConfig.shouldDrop())
+        {
+            ItemStack itemStack = new ItemStack(blockState.getType());
+            BannerMeta bannerMeta = ((BannerMeta) Bukkit.getItemFactory().getItemMeta(blockState.getType()));
```

Comment:
Is that a 1.13 change?

Reply:
Nope. I have yet to go back and reconsider the drop fixes for 1.11, I'll be making a separate PR for that when I have time. For now, the fix is at least in 1.13.